### PR TITLE
added author control type on detail view of post

### PR DESCRIPTION
**Pull request by shahidajmeri786**
Added author control type on detail view of the post

### CODE REVIEW

File: src/system/ui/pages/post/index.js [hunk 1]
```diff
@@ -204,6 +204,15 @@ export const Post = ( { location, match, history } ) => {
 							value: item.date
 
 						},
+						post_author: {
+							label: __( 'Author' ),
+							labelPlacement: 'beside',
+							component: 'select',
+							id: 'post_author',
+							options: item.authorList,
+							value: item.post_author,
```
Comment (fastlinemedia) [inline src/system/ui/pages/post/index.js:213]: Passing a value in the field config doesn't actually do anything. The form hook will pick that up automatically in the `defaults` object that is passed. You can override it there too.

File: src/system/ui/pages/post/index.js [hunk 2]
```diff
@@ -204,6 +204,15 @@ export const Post = ( { location, match, history } ) => {
...
+							isVisible: isAdmin ? true : false
```
Comment (fastlinemedia) [inline src/system/ui/pages/post/index.js:214]: @shahidajmeri786 this isn't showing up for me as `isAdmin` is an empty string, even though I'm logged in as an admin. Can you look into that?

Comment (shahidajmeri786) [reply src/system/ui/pages/post/index.js]: Fixed the issue.

File: src/system/ui/pages/post/index.js (hunk 1 above)
Comment (shahidajmeri786) [reply src/system/ui/pages/post/index.js]: Removed the value field and hookup the item with the default object.

**Comment by fastlinemedia** [comment]
Thanks @shahidajmeri786. This looks great.

**Pull request merged**